whitepapers/business-value/Introduction.md

```diff
@@ -0,0 +1,119 @@
+# What is open source? 
+
+When it comes to open source, there are a lot of misconceptions – even among software engineers. So what exactly is open source? 
+
+Open source software has its roots in the free software movement, which started in the 1970s. Software was the first ‘digital product.’ Long before there were digital music or digital books, you could create a copy of a software program for free, without any impact on the original program. This gave rise to a political philosophy that holds that programmers shouldn’t charge for software, and that software should be a source of empowerment for users. The ‘free’ in the free software movement is a reference to freedom, not to something that costs $0. There are four basic freedoms of the free software movement: 
+
+- Freedom to run the program as you wish, for any purpose
+- The freedom to study how the program works, and change it to make it do what you wish
+- The freedom to redistribute copies so you can help others
+- The freedom to distribute copies of your modified versions to others
+
+The open source software movement takes the philosophy of the free software movement and codifies it, creating a legal framework around free software to make it more palatable to businesses – particularly their legal departments. 
+
+The organization in charge of translating the philosophy of open source software into concrete legal terms is the [Open Source Initiative](https://opensource.org), also known as OSI. All software is released under a software license, and the OSI determines which licenses are open source licenses. You can see the list of OSI-approved licences [here](https://opensource.org/licenses). 
+
+Open source software is simply software that is released under an OSI-approved license. 
+
+It can be published anywhere, it can be created and maintained by volunteers or employees at AWS, it can be used for good and it can be used for evil. It can be produced by a single individual or a group, it can be a product of collaboration between multiple companies or it can be created entirely by a single company. Open source software is not fundamentally ethically or morally superior to proprietary software; it is a development model that can have both advantages and disadvantages. 
```

Comment:
I would not single out one company here. How about we write "hyper-scaler" instead AWS?